src/main/scala/mesosphere/marathon/Main.scala

```diff
@@ -229,8 +229,27 @@ class MarathonApp(args: Seq[String]) extends AutoCloseable with StrictLogging {
 }
 
 object Main {
+  /**
+    * Given environment variables starting with MARATHON_, convert to a series of arguments.
+    *
+    * If environment variable specifies an empty string, treat to boolean flag (no argument)
+    *
+    * @returns A list of args intended to be arg-parsed
+    */
+  def envToArgs(env: Map[String, String]): Seq[String] = {
+    env.flatMap {
+      case (k, v) if k.startsWith("MARATHON_") =>
+        val argName = s"--${k.drop(9).toLowerCase}"
```

Comment:
I like this much more than previous `awk` script 👏

Reply:
Right? I spent 2 hours trying to fix the awk script and I then I thought, "why am I doing this lol?